src/services/api.ts

```diff
@@ -156,6 +158,21 @@ const updateUser = async (token: string, data: Profile): Promise<Profile> => {
   return response.data
 }
 
+const createSubscriptionIdOneSignal = async (token: string, subscriptionId: string): Promise<{}> => {
+  const url = BACKEND_URL + '/onesignal/'
+  const headers = { Authorization: `Token ${token}` }
+  const data = { subscription_id: subscriptionId }
+  const response = await axios.post<OneSignal>(url, data, { headers })
+  return response.data
+}
+
+const getSubscriptionIdOneSignal = async (token: string): Promise<SubscriptionInfo> => {
```

Comment:
`getOneSignalSubscriptionId` feels more natural. Same thing for the above function

Reply:
I Updated